frontend/src/types/search/searchResponseTypes.ts

```diff
@@ -64,16 +64,19 @@ export interface SearchAPIResponse extends APIResponse {
   fieldChanged?: string;
 }
 
+export const validSearchQueryParamKeys = [
```

Comment:
wanted to be able to use these values for iteration, so switched up a bit of the typing here